test/libsolidity/semanticTests/array/copying/array_copy_storage_storage_static_static.sol

```diff
@@ -13,5 +13,7 @@ contract c {
     }
```

Comment:
I wasn't aware that assignments between static arrays of different sizes were even allowed. I don't see it mentioned in the `Arrays` section of the doc. Perhaps it should say something about it there?

Reply:
That is very good point. I wasn't also aware of this until I saw this issue. I'll document this as well.